local-cli/generator-android/templates/src/app/proguard-rules.pro

```diff
@@ -40,10 +40,13 @@
 
 -keep class * extends com.facebook.react.bridge.JavaScriptModule { *; }
 -keep class * extends com.facebook.react.bridge.NativeModule { *; }
+-keep class * { native <methods>; }
```

Comment:
I'm not very familiar with this syntax. Can you please add a comment what this does?